rust/osm2lanes/src/locale.rs

```diff
@@ -16,7 +16,7 @@ impl Locale {
     pub fn builder() -> Config {
         Config::default()
     }
-    pub fn default_width(&self, designated: &LaneDesignated) -> Metre {
+    pub const fn travel_width(&self, designated: &LaneDesignated) -> Metre {
```

Comment:
Isn't it likely we'll break `const` in the near future? If locale starts doing any logic to use config for different places, we might need something simple, even a `HashMap`, that isn't const yet

Reply:
```suggestion
    pub fn travel_width(&self, designated: &LaneDesignated) -> Metre {
```